src/acceptance-test/java/org/zalando/nakadi/webservice/EventStreamReadingAT.java

```diff
@@ -327,6 +334,52 @@ private Response readEvents() {
     }
 
     @Test(timeout = 10000)
+    public void whenExceedMaxConsumersNumThen429() throws IOException, InterruptedException, ExecutionException,
```

Comment:
@v-stepanov it would be good to have a test case for when a connection is closed. It should release one slot.